network/network-tutorial-standalone.md

```diff
@@ -436,6 +436,10 @@ connected to both networks.
     address. This capability is called **automatic service discovery**. Let's
     connect to `alpine1` and test this out. `alpine1` should be able to resolve
     `alpine2` and `alpine4` (and `alpine1`, itself) to IP addresses.
+    
+    > **Note**
+    > 
+    > Only valid for custom container names, the default generated container names cannot be resolved
```

Comment:
Thanks @wushu037 but I think the paragraph above already covers this?

> On user-defined networks like alpine-net, containers can not only communicate by IP address, but can also resolve a container name to an IP address. This capability is called automatic service discovery. Let's connect to alpine1 and test this out. alpine1 should be able to resolve alpine2 and alpine4 (and alpine1, itself) to IP addresses.

But let me know if you think that "user-define" part needs clarifying and maybe we could move the note into that line instead?

Reply:
Thank you for your reply.

I don't think the paragraph you mentioned covers this, it only highlights "user-defined networks" and I added "user-defined container names".